docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -0,0 +1,104 @@
+# Deploy to Fly.io with SQLite
+
+:::note
+
+To follow this how-to guide, you'll first need to install the Fly CLI and create
+an account by [following this official guide](https://fly.io/docs/hands-on/).
+
+:::note
+
+Navigate to your Platformatic DB project in the terminal on your local machine.
+Run `fly launch` and follow the prompts. When it asks if you want to deploy
+now, say "no" as there are a few things that you'll need to configure first.
+
+<!-- TODO: Could probably put a command here to launch the app in non-interactive mode -->
+
+Create a volume for database storage:
+
+```bash
+fly volumes create data
+```
+
+This will create storage in the same region as the application. The volume
+defaults to 3GB size, use  `-s` to change the size. For example, `-s 10` is 10GB.
+
+Add a `mounts` section in **fly.toml**, replacing `<app-name>` with the name
+of your application:
+
+<!-- TODO: Check this mount path, probably not correct now we're not using Docker -->
+
+```toml
+[mounts]
+  source = "data"
+  destination = "/opt/<app-name>/.platformatic/data"
```

Comment:
```suggestion
  destination = "/app/.platformatic/data"
```

Good catch. The buildpack used by fly puts it into `/app`. We'll follow the same pattern in the future.